winrt/lib/text/CanvasFontFace.cpp

```diff
@@ -666,6 +666,114 @@ IFACEMETHODIMP CanvasFontFace::GetGlyphIndices(
         });
 }
 
+IFACEMETHODIMP CanvasFontFace::GetGlyphMetrics(
+  uint32_t inputCount,
+  int* inputElements,
+  boolean isSideways,
+  uint32_t* outputCount,
+  Rect** outputElements)
+{
+  return ExceptionBoundary(
+    [&]
+  {
```

Comment:
This is more minor, but it appears that you have your editor set to two spaces. In the code, we have a standard of treating tabs as four spaces. Would you be able to make this new code consistent with the rest?
Admittedly, we have some places in our code that violate this rule. However, it's the convention.